sql/ir_generator.go

```diff
@@ -142,13 +142,30 @@ func parseExpression(e interface{}) (interface{}, error) {
 	return nil, fmt.Errorf("not supported expr: %v", el)
 }
 
+func inferStringValue(expr string) interface{} {
+	ret, err := strconv.Atoi(expr)
+	if err == nil {
+		return ret
+	}
+	retFloat, err := strconv.ParseFloat(expr, 32)
+	if err == nil {
+		// NOTE: always use float32 for attributes, we may never use a
```

Comment:
`NOTE` => `NOTE(typhoonzero)`. It is better to indicate the owner of the note.

Reply:
`golint` seems is not checking `// NOTE` comments, so I just removed `NOTE` here.